packages/mds-web-sockets/server.ts

```diff
@@ -2,12 +2,17 @@ import logger from '@mds-core/mds-logger'
 import { seconds, getEnvVar } from '@mds-core/mds-utils'
 import WebSocket from 'ws'
 import { setWsHeartbeat } from 'ws-heartbeat/server'
-import { Telemetry, VehicleEvent } from '@mds-core/mds-types'
 import { ApiServer, HttpServer } from '@mds-core/mds-api-server'
 import { initializeNatsSubscriber } from '@mds-core/mds-stream/nats/nats'
 import { Clients } from './clients'
 import { ENTITY_TYPE } from './types'
 
+type Json = boolean | number | string | null | JsonArray | JsonMap
```

Comment:
This should get moved to a better place if agreement on approach

Reply:
There are some similar types in `@mds-core/mds-repository` ... I would be in favor of removing the types from `mds-web-sockets` and moving the definitions from `mds-repository` to `mds-types`
```ts
// eslint-reason recursive declarations require interfaces
// eslint-disable-next-line @typescript-eslint/no-empty-interface
export interface JsonArray extends Array<Json> {}

export interface JsonObject {
  [property: string]: Json
}

export type JsonValue = string | number | boolean | JsonArray | JsonObject

export type Json = Nullable<JsonValue>
```

Reply:
Done